pkg/curatedpackages/bundlesource.go

```diff
@@ -0,0 +1,32 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"strings"
+)
+
+type BundleSource string
+
+const (
+	Cluster  = "cluster"
+	Registry = "registry"
+)
+
+func (b BundleSource) String() string {
+	return string(b)
+}
+
+func (b *BundleSource) Set(s string) error {
```

Comment:
Could we add a doc comment to these methods explaining that they implement an interface required by cobra for validation?